go/fn/examples/example_validator_test.go

```diff
@@ -3,26 +3,26 @@ package example
 import (
 	"os"
 
-	"github.com/GoogleContainerTools/kpt-functions-sdk/krmfn"
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
 )
 
 // This example implements a function that validate resources to ensure
 // spec.template.spec.securityContext.runAsNonRoot is set in workload APIs.
 
 func Example_validator() {
-	if err := krmfn.AsMain(krmfn.ResourceListProcessorFunc(validator)); err != nil {
+	if err := fn.AsMain(fn.ResourceListProcessorFunc(validator)); err != nil {
 		os.Exit(1)
 	}
 }
 
-func validator(rl *krmfn.ResourceList) error {
-	var results krmfn.Results
+func validator(rl *fn.ResourceList) error {
+	var results fn.Results
 	for _, obj := range rl.Items {
 		if obj.APIVersion() == "apps/v1" && (obj.Kind() == "Deployment" || obj.Kind() == "StatefulSet" || obj.Kind() == "DaemonSet" || obj.Kind() == "ReplicaSet") {
 			var runAsNonRoot bool
 			obj.GetOrDie(&runAsNonRoot, "spec", "template", "spec", "securityContext", "runAsNonRoot")
```

Comment:
I have seen of pattern of using `obj.MustGet(...)` or `obj.MustSet(...)` in Go libraries for things that panics.

Reply:
I'll close this comment now becuase this part would be changed soon. The panic handling is one thing I'll do more research on.